data/paradise world jobs.txt

```diff
@@ -403,3 +403,89 @@ mission "Pleasure cruise security"
 		dialog "You bid goodbye to the captain of the <npc> and accept your payment of <payment>."
 		payment
 		payment 50000
+
+# Brig bulk missions
```

Comment:
Preserve the whitespace gaps between disparate mission types (you only have a single line of separation).

Reply:
Should be fixed now